### ISSUE TITLE
Can we run it on NVdia discrete graphics-1060/80?

### ISSUE THREAD

**Issue by nicks165** [TLESORT/YOLO-TensorRT-GIE-#6]
When i run in 1060 i have this issue 

In file included from YOLODetector.cpp:22:0:
YOLODraw.h:30:12: error: ‘Dims3’ in namespace ‘nvinfer1’ does not name a typenvinfer1::Dims3 dims

**Comment by Allen-Shao** [comment]
Yes, you can run it on GTX 1060/80. Generally you can use TensorRT on any Nvidia GPU supporting CUDA. 
The problem you have is because `Dims3` is deprecated in current TensorRT. Simply change it to `Dims` should solve the problem.

**Comment by molyswu** [comment]
Dims3   change it to Dims
